src/tests/Builder.test.js

```diff
@@ -0,0 +1,594 @@
+      AbsoluteMaximumProvisioned: 8999,
+      MinutesOfStatsToRetrieve: 8998,
+      MinutesOfStatsToIgnore: 8997,
+   };
+
+   wildcardResourceConfig = {
+      MinutesOfStatsToRetrieve: 7999,
+      MinutesOfStatsToIgnore: 7998,
+   };
+
+   specificResourceConfig = {
+      MinutesOfStatsToIgnore: 6999,
+   };
+
+   expect(customUserConfig.AbsoluteMinimumProvisioned).to.not.eql(defaultConfig.AbsoluteMinimumProvisioned);
+   expect(customUserConfig.AbsoluteMaximumProvisioned).to.not.eql(defaultConfig.AbsoluteMaximumProvisioned);
+   expect(customUserConfig.MinutesOfStatsToRetrieve).to.not.eql(defaultConfig.MinutesOfStatsToRetrieve);
+   expect(customUserConfig.MinutesOfStatsToIgnore).to.not.eql(defaultConfig.MinutesOfStatsToIgnore);
+   expect(customUserConfigForType.AbsoluteMaximumProvisioned).to.not.eql(defaultConfig.AbsoluteMaximumProvisioned);
+   expect(customUserConfigForType.MinutesOfStatsToRetrieve).to.not.eql(defaultConfig.MinutesOfStatsToRetrieve);
+   expect(customUserConfigForType.MinutesOfStatsToIgnore).to.not.eql(defaultConfig.MinutesOfStatsToIgnore);
+   expect(wildcardResourceConfig.MinutesOfStatsToRetrieve).to.not.eql(defaultConfig.MinutesOfStatsToRetrieve);
+   expect(wildcardResourceConfig.MinutesOfStatsToIgnore).to.not.eql(defaultConfig.MinutesOfStatsToIgnore);
+   expect(specificResourceConfig.MinutesOfStatsToIgnore).to.not.eql(defaultConfig.MinutesOfStatsToIgnore);
+
+   beforeEach(function() {
+      builder = new Builder();
+   });
+
+   describe('resource exclusion', function() {
+
+      it('excludes a particular resource by name - tables', function() {
+         builder.excludeTable(tbl1R.tableName);
+         expect(builder.isExcludedResource(tbl1R)).to.be(true);
+         expect(builder.isExcludedResource(tbl1W)).to.be(true);
+         expect(builder.isExcludedResource(tbl1idR)).to.be(false);
+         expect(builder.isExcludedResource(tbl1idW)).to.be(false);
+         expect(builder.isExcludedResource(tbl2R)).to.be(false);
+         expect(builder.isExcludedResource(tbl2W)).to.be(false);
+         expect(builder.isExcludedResource(tbl2idR)).to.be(false);
+         expect(builder.isExcludedResource(tbl2idW)).to.be(false);
+      });
+
+      it('excludes a particular resource by name - indexes', function() {
+         builder.excludeIndex(tbl1idR.tableName, tbl1idR.indexName);
+         expect(builder.isExcludedResource(tbl1R)).to.be(false);
+         expect(builder.isExcludedResource(tbl1W)).to.be(false);
+         expect(builder.isExcludedResource(tbl1idR)).to.be(true);
+         expect(builder.isExcludedResource(tbl1idW)).to.be(true);
+         expect(builder.isExcludedResource(tbl2R)).to.be(false);
+         expect(builder.isExcludedResource(tbl2W)).to.be(false);
+         expect(builder.isExcludedResource(tbl2idR)).to.be(false);
+         expect(builder.isExcludedResource(tbl2idW)).to.be(false);
+      });
+
+      it('excludes multiple resources by using wildcard in name - tables', function() {
```

Comment:
This description seems wrong. I think you meant to say `it('still excludes an individual table when there is a wildcard at the end of the table name' ...`) or something like that?